ceiling-tests/v2/README.md

```diff
@@ -0,0 +1,606 @@
+
+Apart from the raw numbers in the previous table, running these tests revealed some key points:
+
+- The main bottleneck we see across most of the tests is CPU.
+- Elasticsearch: Ceiling Tests v2 is the first time we add Elasticsearch to a batch of performance tests in a structured way. We learned some interesting things:
+  - Elasticsearch improved performance by an order of magnitude in the lower-end tests, when the main bottleneck was the DB. This proves that Elasticsearch is the best solution when the database is struggling.
+  - However, ES did absolutely nothing in the high-end tests, that were always bound by CPU.
+  - Hence, for Elasticsearch to improve stuff, the bottleneck *has to be* in the database.
+  - We used 2 nodes with a fixed instance type for all tests. The CPU of the instances maxed out starting with ~30k users. Thus, the potential improvement attributed to Elasticsearch is actually lost in tests with more than 30k users. We need to scale it, either vertically or horizontally, along with the expected number of supported users to see an improvement. We need more tests to provide a solid answer to this.
+- Network:
+  - High bandwidth is definitely a bottleneck in higher scales. This happens in micro-bursts, not in the average usage of the network, which comfortably sits below the thresholds set by the instance types used in the tests. However, due to usage peaks, we had to bump the originally planned proxy instance type for it to be able to handle the high load.
+  - Intra-cluster communication started to show some signs of deterioration after ~90k with 5 app nodes. It isn't a bottleneck at all for now, but it may be soon. Increasing the app types or even the number of app nodes may help, because the bottleneck here was network and the hardware wasn't able to cope with it. We need more tests to provide a solid answer to this.
+
+Based on these learnings, we have a series of planned improvements:
+
+- Immediate steps:
+  - Add `ethtool` stats to the Grafana dashboards ([MM-59994](https://mattermost.atlassian.net/browse/MM-59994)): this will help us monitor not only the average bandwidth used, but the potential micro-bursts resulting in dropped packets.
+  - Use network-optimized instances (the *n.* series) for both proxy and app nodes: this will help remove the bottleneck in the network.
+  - Surface AWS internal metrics via Cloudwatch ([MM-60020](https://mattermost.atlassian.net/browse/MM-60020)): this will help us monitor AWS-controlled instances, like the database, Elasticsearch or Redis.
+- Big steps ahead:
+  - Improve intra-cluster communication ([MM-58564](https://mattermost.atlassian.net/browse/MM-58564)): this will be needed as we scale further, although it is not a bottleneck right now.
+  - Multi-proxy setups: scaling further with only one entry point seems to be unsustainable, so we need to investigate setups with more than one proxy.
+  - In-depth investigation on CPU: all tests in the higher-end are 
```

Comment:
This seems to be cut

Reply:
Whoops, good catch. Fixed in https://github.com/mattermost/performance-reports/pull/22/commits/f43d0e41ffb270757875997d5d396b28701f6a75.